src/components/RJST/Lenses/types/table.tsx

```diff
@@ -12,16 +12,15 @@ import {
 import { useTagActions } from '../../components/Table/useTagActions';
 import { AddTagHandler } from '../../components/Table/AddTagHandler';
 import { Box, styled, Tooltip, Typography } from '@mui/material';
-import { color } from '@balena/design-tokens';
 import { Copy } from '../../../Copy';
 import type { RJSTEntityPropertyDefinition } from '../..';
 import { useAnalyticsContext } from '../../../../contexts/AnalyticsContext';
 
-const TagContainer = styled(Box)(() => ({
-	border: `0.5px solid ${color.border.accent.value}`,
-	borderRadius: '3px',
-	color: color.text.value,
-	backgroundColor: color.bg.accent.value,
+const TagContainer = styled(Box)(({ theme }) => ({
+	border: `0.5px solid ${theme.vars.palette.border.accent.value}`,
+	borderRadius: 'sm',
```

Comment:
I see you have changed `8px` to `sm` in other places and `3px` to `sm` here. Just want to make sure the differences from current are intentional

Reply:
good catch, 8px is supposed to be `md`